app/javascript/gobierto_data/webapp/components/sets/SummaryTab.vue

```diff
@@ -61,6 +62,10 @@ export default {
       required: true,
       default: () => [],
     },
+    isUserLogged: {
```

Comment:
Do you see what annoying is the children props propagation, huh? :D That's why I tell you about the `v-slot`, containers and so...